airflow/contrib/operators/bigquery_export_query_results_to_gcs.py

```diff
@@ -0,0 +1,138 @@
+from airflow.utils.decorators import apply_defaults
+from airflow.models import BaseOperator
+from airflow.plugins_manager import AirflowPlugin
+from airflow.contrib.hooks.bigquery_hook import BigQueryBaseCursor, BigQueryHook
+from airflow.exceptions import AirflowException
+from uuid import uuid4
+import logging
+
+
+class BigQueryExportQueryResultsToGCS(BaseOperator):
+    """
+    Executes a BigQuery Standard SQL query and exports query results to GCS.
+    For large query results, a GCS * pattern must be present
+    Grant necessary Service Account permission to your output GCS location table
+
+    :param project_id: The GCP project where the query gets executed, temp table is stored and extracted from
+    :type project_id: string
+    :param query: The BigQuery SQL to execute.
+    :type query: string
+    :param destination_cloud_storage_uris: The destination Google Cloud
+        Storage URI (e.g. gs://some-bucket/some-file.txt). (templated) Follows
+        convention defined here:
+        https://cloud.google.com/bigquery/exporting-data-from-bigquery#exportingmultiple
+    :type destination_cloud_storage_uris: List[str]
+    :param compression: Type of compression to use.
+    :type compression: str
+    :param export_format: File format to export.
```

Comment:
You take export_format as parameter but it's never being used.
You hard codded CSV later as the only option

Reply:
this is fixed now